<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>snappingClass</class>
 <widget class="QMainWindow" name="snappingClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>snapping</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="my_qlabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>561</width>
      <height>311</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionSketch_Snapping"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionDraw"/>
   <addaction name="actionSphere"/>
   <addaction name="actionCylinder"/>
   <addaction name="actionCone"/>
   <addaction name="actionForward"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="snapping.qrc">
     <normaloff>:/snapping/folder.png</normaloff>:/snapping/folder.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSphere">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="snapping.qrc">
     <normaloff>:/snapping/sphere.png</normaloff>:/snapping/sphere.png</iconset>
   </property>
   <property name="text">
    <string>sphere</string>
   </property>
  </action>
  <action name="actionCylinder">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="snapping.qrc">
     <normaloff>:/snapping/cylinder.png</normaloff>:/snapping/cylinder.png</iconset>
   </property>
   <property name="text">
    <string>cylinder</string>
   </property>
  </action>
  <action name="actionCone">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="snapping.qrc">
     <normaloff>:/snapping/cone.png</normaloff>:/snapping/cone.png</iconset>
   </property>
   <property name="text">
    <string>cone</string>
   </property>
  </action>
  <action name="actionForward">
   <property name="icon">
    <iconset resource="snapping.qrc">
     <normaloff>:/snapping/forward.png</normaloff>:/snapping/forward.png</iconset>
   </property>
   <property name="text">
    <string>forward</string>
   </property>
  </action>
  <action name="actionSketch_Snapping">
   <property name="text">
    <string>Sketch Snapping</string>
   </property>
  </action>
  <action name="actionDraw">
   <property name="icon">
    <iconset resource="snapping.qrc">
     <normaloff>:/snapping/design.png</normaloff>:/snapping/design.png</iconset>
   </property>
   <property name="text">
    <string>draw</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>my_qlabel</class>
   <extends>QLabel</extends>
   <header location="global">my_qlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="snapping.qrc"/>
 </resources>
 <connections/>
</ui>
